### ISSUE TITLE
Plotting one heat map for topics in two topic models

### ISSUE THREAD

**Issue by Hari25483** [MaartenGr/BERTopic#778]
Hi @MaartenGr 
Usually, heat map is plotted for one topic model with its topics in both axis. 
I want to plot heatmap by finding cosine similarity of topics from two models, such that topics from one model on one axis, and topics of other model on other axis.
Are there any methods to do this?

**Comment by MaartenGr** [comment]
Yes, there is actually some [documentation](https://maartengr.github.io/BERTopic/getting_started/tips_and_tricks/tips_and_tricks.html#finding-similar-topics-between-models) on this that could be helpful. In practice, you would need to do something like the following:

```python
import plotly.express as px
from sklearn.metrics.pairwise import cosine_similarity

# Calculate a distance matrix
distance_matrix = cosine_similarity(topic_model_one.topic_embeddings_, topic_model_two.topic_embeddings_)

# Plot the heatmap
fig = px.imshow(
    distance_matrix, 
    labels=dict(color="Similarity Score"), 
    y=list(topic_model_one.topic_labels_.values()), 
    x=list(topic_model_two.topic_labels_.values()), 
    color_continuous_scale='GnBu'
)
fig
```

**Comment by Hari25483** [comment]
Thank you so much @MaartenGr .
